src/Connector.js

```diff
@@ -317,6 +317,11 @@ class Connector {
     return { id: device.id, token: device.id };
   }
 
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Do you really need to disable this `eslint` rule here?

Reply:
Yeah, we don't use the `token` parameter :disappointed:

If I remove it, the linter blocks me from committing with the following error: `error  'token' is defined but never used  no-unused-vars`.

Reply:
Ack.